Xamarin.Forms.Core.UITests.Shared/Utilities/Gestures.cs

```diff
@@ -7,6 +7,24 @@ namespace Xamarin.Forms.Core.UITests
 {
 	internal static class Gestures
 	{
+		static AppRect GetAppRect(this IApp app, Func<AppQuery, AppQuery> elementQuery)
```

Comment:
Might consider refactoring to `QueryNTimes` that takes an additional parameter `count`. Then pull `10` into a const that's passed to at each callsite.